app/components/list-uploaded-assets/ListUploadedAssets.tsx

```
@@ -0,0 +1,86 @@
+              <th className="border border-slate-600 px-4 py-1">Created</th>
+              <th className="border border-slate-600 px-4 py-1">Updated</th>
+              <th className="border border-slate-600 px-4 py-1">Status</th>
+            </tr>
+          </thead>
+          <tbody>
+            {filteredCreatorAssets.map((video, i) => (
+              <UploadAsset
+                activeAccount={props.activeAccount}
+                asset={video}
+                idx={i}
+                key={i + '-' + video.id}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)